src/modals/CssSnippetFuzzySuggestModal.ts

```diff
@@ -46,10 +61,24 @@ export class CssSnippetFuzzySuggestModal extends FuzzySuggestModal<
 				command: Platform.isMacOS ? "⌘ del" : "ctrl del",
 				purpose: "to delete",
 			},
+			{ command: "tab", purpose: "to toggle state" },
```

Comment:
```suggestion
			{ command: "tab", purpose: "to enable/disable" },
```
"to toggle state" is generic, it's not clear what state you are toggling. I have a suggestion, feel free to come up with something else or keep this, I won't block this change if you leave it as is.

Reply:
I had that initially, but it the instructions then changed to two lines. I can change it again

Reply:
We do have quite a lot of instructions that will start wrapping depending on screen size, I'll think on this